packages/runtime-common/tests/query-test.ts

```diff
@@ -20,6 +20,30 @@ interface TestCards {
 }
 
 const tests = Object.freeze({
+  'can get prerendered cards': async (
+    assert,
+    { indexer, loader, testCards },
+  ) => {
+    let { mango, vangogh } = testCards;
+    await setupIndex(indexer, [
+      {
+        card: mango,
+        data: { embedded_html: { default: '<div>Mango</div', deps: 'abc' } },
+      },
+      {
+        card: vangogh,
+        data: { embedded_html: { default: '<div>Mango</div' } },
+      },
+    ]);
+
+    let { prerenderedCards: _prerenderedCards, meta } =
+      await indexer.searchPrerendered(new URL(testRealmURL), {}, loader);
+
+    // TODO: how to test when there is no deps for instances, and no indexed modules in boxel index?
```

Comment:
So in this case (testing in browser with sqlite), the instances in the boxel index table will have no `deps` populated, and there will be no `css` entries since we have shimmed modules, which means the query that searches for prerendered cards will not find any css in the table. I'm unsure what would be the right thing to do here–do we want to manually set `deps` and add `css` entries?

Reply:
in this case since this is a low level unit test, yes, you would manually set the css and deps. This is _not_ testing what current-run does. this is only testing what the indexer performs. It is the job of current-run.ts to set the css and deps. specifically in the host/tests/integratino/search-index-test.ts is where we test current-run. In this PR we did not change how deps are created nor did we change how css is created. there are other tests for that. If you want to put together a test that goes thru the whole flow, that is fine, but it would have to live in the test module I just mentioned, and it would have to use string module source like we discussed in the meeting (we actually test css that way in that particular test module).